### DISCUSSION TITLE
Security

### DISCUSSION THREAD

**Discussion by Matthew-Wise** [TimGeyssens/Umbraco-v8-Community-Website-StarterKit#19] [General]
Make sure we are promoting good and secure practices.

One tool for checking would be - https://observatory.mozilla.org/

**Comment by TimGeyssens** [comment]
Yes! Just stumbled upon this web.config best practices setup: https://github.com/h5bp/server-configs-iis/blob/master/dotnet%204.5/MVC5/Web.config (security is part of it)

**Comment by louisjrdev** [comment]
Also maybe a MFA implementation too?

**Reply by TimGeyssens** [reply]
for backoffice? Good idea!

**Reply by louisjrdev** [reply]
Yeah! Hopefully encourages more people to implement it :+1:

**Reply by TimGeyssens** [reply]
good stuff, yeah! Will add it to tracker once I start setting up the tasks
